src/common/utxobased/keymanager/keymanager.ts

```diff
@@ -899,15 +899,10 @@ export function pubkeyToScriptPubkey(
 }
 
 function getMaxLengthOfCoinPrefixes(coin: CoinPrefixes): number {
-  let max = 0
-  for (const value of Object.values(coin)) {
-    if (value instanceof Array) {
-      if (max < value.length) {
-        max = value.length
-      }
-    }
-  }
-  return max
+  return Object.values(coin).reduce(
```

Comment:
Since it's just a reduce, do we really even to make it a function?

Reply:
I'll check and see. Good point.

Reply:
Inlined it